config/routes.rb

```diff
@@ -30,6 +30,8 @@
 
   resources :custom_manifests, only: [:create, :update, :destroy, :show]
 
+  resources :tool_groups, only: [:create, :destroy, :index, :show, :update]
```

Comment:
On stage the API is currently responding at `/tool_groups` and not `/tool-groups`, I think this will fix that

```suggestion
  resources :tool_groups, path: "tool-groups", only: [:create, :destroy, :index, :show, :update]
```